tools: add pdctl download guide (#1710)
pingcap/docs: dev/reference/tools/pd-control.md

>
> Generally, you don't need to compile source code as the PD Control tool already exists in the released Binary or Docker. However, dev users can refer to the above instruction for compiling source code.

## Download TiDB installation package

If you want to download the latest version of `pd-ctl`, directly download the TiDB package, because `pd-ctl` is included in the TiDB package.

> | Package name | OS | Architecture | SHA256 checksum |

Reviewer: ```suggestion
| Package download link | OS | Architecture | SHA256 checksum |
```

|:---|:---|:---|:---|
| `http://download.pingcap.org/tidb-{version}-linux-amd64.tar.gz` (pd-ctl) | Linux | amd64 | `http://download.pingcap.org/tidb-{version}-linux-amd64.sha256` |

> **Note:**
>

> > `{version}` in the above download link indicates the version number of TiDB. For example, the download link for `v3.0.7` is `http://download.pingcap.org/tidb-v3.0.7-linux-amd64.tar.gz`. You can also download the latest unpublished version by replacing `{version}` with `latest`.

Reviewer: ```suggestion
> `{version}` indicates the version number of TiDB. For example, if `{version}` is `v3.0.7`, the package download link is `http://download.pingcap.org/tidb-v3.0.7-linux-amd64.tar.gz`. You can also download the latest unpublished version by replacing `{version}` with `latest`.
```
